client/ayon_core/hosts/substancepainter/plugins/load/load_mesh.py

```diff
@@ -11,7 +12,15 @@
 from ayon_core.hosts.substancepainter.api.lib import prompt_new_file_with_mesh
 
 import substance_painter.project
-import qargparse
+
+
+def get_uv_workflow(uv_option="default"):
+    if uv_option == "default":
+        return substance_painter.project.ProjectWorkflow.Default
+    elif uv_option == "uvTile":
+        return substance_painter.project.ProjectWorkflow.UVTile
+    else:
+        return substance_painter.project.ProjectWorkflow.TextureSetPerUVTile
```

Comment:
Let's be more explicit.
```suggestion
def get_uv_workflow(uv_option="default"):
    if uv_option == "default":
        return substance_painter.project.ProjectWorkflow.Default
    elif uv_option == "uvTile":
        return substance_painter.project.ProjectWorkflow.UVTile
    elif uv_option == "textureSetPerUVTile":
        return substance_painter.project.ProjectWorkflow.TextureSetPerUVTile
    else:
        raise ValueError(f"Unsupported UV option: {uv_option}")
```